### PR TITLE
Install missing packages required by vagrant to talk to Windows hosts

### PR DESCRIPTION
**Pull request by netoarmando**
Install rubygem-winrm, rubygem-winrm-fs and winrm-elevated packages from
a temporary COPR repo.

Signed-off-by: Armando Neto <abiagion@redhat.com>

### CODE REVIEW

**Comment by netoarmando** [comment]
Tested on a fedora 30 instance: http://freeipa-org-pr-ci.s3-website.eu-central-1.amazonaws.com/jobs/e990b0f6-29aa-11ea-a5ec-fa163e3dbfd5/

File: ansible/roles/runner/tasks/setup.yml [hunk 1]
```diff
@@ -73,6 +73,22 @@
   when: vagrant_version.stdout is version('2.2.0', '<')
   ignore_errors: yes
 
+# TODO: Replace COPR repo when the packages hit the stable repos
+- name: install winrm packages and dependencies for AD tests
+  block:
+    - name: enable winrm copr repo
+      shell: "dnf copr enable -y abiagion/winrm"
```
Comment (flo-renaud) [inline ansible/roles/runner/tasks/setup.yml]: Instead of using a personal copr repo, @freeipa/freeipa-master could be leveraged.

Comment (netoarmando) [reply ansible/roles/runner/tasks/setup.yml]: Nice, I've requested `builder` and `admin` permissions to that repo.

Comment (netoarmando) [reply ansible/roles/runner/tasks/setup.yml]: Packages added to [@freeipa/freeipa-pr-ci](https://copr.fedorainfracloud.org/coprs/g/freeipa/freeipa-pr-ci/packages/) repo.

**Comment by netoarmando** [comment]
@f-trivino do you acknowledge these changes? May I merge it?

**Comment by f-trivino** [comment]
> @f-trivino do you acknowledge these changes? May I merge it?

yes. Let's merge it.

**Pull request merged**